docs/rule/no-heading-inside-button.md

```diff
@@ -0,0 +1,30 @@
+## no-heading-inside-button
+
+Assistive technology allows users to browse a page by heading elements (h1-h6). However, if those heading elements are nested inside of button elements, they will automatically be marked as presentational by browsers. Any HTML element where ["children presentational" is true](https://w3c.github.io/aria/#button) should be coerced by the browser to be presentational, and therefore not included in the accessibility tree. 
```

Comment:
> if those heading elements are nested inside of button elements, they will automatically be marked as presentational by browsers

1) can you add a link that states this?
2) does this apply to "heading inside of buttons" or does it apply to "heading inside of interactive elements"?